### PR TITLE
ENH: CP and Thrust Eccentricity Effects Generate Roll Moment

### PR DESCRIPTION
**Pull request by MateusStano**
## Pull request type

- [x] Code changes (bugfix, features)

## Checklist

- [x] Tests for the changes have been added (if needed)
- [x] Docs have been reviewed and added / updated
- [x] Lint (`black rocketpy/ tests/`) has passed locally 
- [x] All tests (`pytest tests -m slow --runslow`) have passed locally
- [x] `CHANGELOG.md` has been updated (if relevant)

## Current behavior

CP eccentricities did not generate moment on the rocket's center axis (roll moment M3) and thrust eccentricity was not included in `u_dot_generalized`

## New behavior

Roll moment is calculated properly and thrust eccentricity is included in `u_dot_generalized`

## Breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Gui-FernandesBR — COMMENTED** [review]
Send like r1 and r2 are aligned to the global coordinate system (the earth) but the x and y excentricity should be aligned to the local (rocket) coordinate system.

Right now I understand you are assuming R1 is always parallel to x_excentricity, right?

**Review by Gui-FernandesBR — COMMENTED** [review]
Please update the CHANGELOG

**Comment by MateusStano** [comment]
> Send like r1 and r2 are aligned to the global coordinate system (the earth) but the x and y excentricity should be aligned to the local (rocket) coordinate system.
> 
> Right now I understand you are assuming R1 is always parallel to x_excentricity, right?

R1, R2, and R3 are described in the rocket coordinate system from what I understand

What I did here is in accordance with what the code was doing before, I am not assuming anything different. But yes, I believe the R1 was assumed to be parallel to x

File: tests/unit/test_rocket.py [hunk 1]
```diff
@@ -496,3 +496,26 @@ def test_get_inertia_tensor_derivative_at_time(calisto):
     assert pytest.approx(0, atol) == inertia_tensor.y[2]
     assert pytest.approx(0, atol) == inertia_tensor.z[0]
     assert pytest.approx(0, atol) == inertia_tensor.z[1]
+
+
+def test_add_cp_eccentricity(calisto):
+    """Test add_cp_eccentricity method of the Rocket class."""
+    calisto.add_cp_eccentricity(0.1, 0.1)
+    assert calisto.cp_eccentricity_x == 0.1
+    assert calisto.cp_eccentricity_y == 0.1
```
Comment (Gui-FernandesBR) [inline tests/unit/test_rocket.py:505]: This test already exists here? https://github.com/RocketPy-Team/RocketPy/blob/54c69a323714b22404c9033f019892f4b0dcdc59/tests/unit/test_rocket.py#L345

**Review by Gui-FernandesBR — APPROVED** [review]

File: tests/unit/test_rocket.py (hunk 1 above)
Comment (Gui-FernandesBR) [reply tests/unit/test_rocket.py]: [36622ad](https://github.com/RocketPy-Team/RocketPy/pull/617/commits/36622adced8e89e9adbf59eec19ec9fb3a90d2e2)

**Pull request merged**